src/rp2_common/pico_cyw43_driver/include/pico/cyw43_driver.h

```diff
@@ -58,9 +58,13 @@ void cyw43_driver_deinit(struct async_context *context);
 #endif
 #endif
 
-// PICO_CONFIG: CYW43_PIO_CLOCK_DIV_FRAC, Fractional part of the clock divider for communication with the wireless chip, type=bool, default=0, group=pico_cyw43_driver
-#ifndef CYW43_PIO_CLOCK_DIV_FRAC
-#define CYW43_PIO_CLOCK_DIV_FRAC 0
+// PICO_CONFIG: CYW43_PIO_CLOCK_DIV_FRAC8, Fractional part of the clock divider for communication with the wireless chip 0-255, type=bool, default=0, group=pico_cyw43_driver
```

Comment:
If we're renaming the `PICO_CONFIG:` entry from `CYW43_PIO_CLOCK_DIV_FRAC` to `CYW43_PIO_CLOCK_DIV_FRAC8`, we probably ought to do the same for the `PICO_CMAKE_CONFIG:` entry at https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2_common/pico_cyw43_driver/CMakeLists.txt#L87-L90

Reply:
good catch; @peterharperuk since this value was new post SDK2.0 I'm not going to support the old value

Reply:
presumably much use OF it?

Reply:
yes, there's unlikely to be much need for changing the cyw43 pio divider by anything but a whole number.